templates/shared/variables/variable_list.tmpl

```diff
@@ -0,0 +1,97 @@
+<h4 class="ui top attached header">
+	{{.locale.Tr "actions.variables.management"}}
+	<div class="ui right">
+		<button class="ui primary tiny show-modal show-variable-edit-modal button"
+			data-is-new="true"
+			data-base-action="{{.Link}}/new"
+			data-modal="#edit-variable-modal"
+			data-modal-header="{{.locale.Tr "actions.variables.creation"}}"
+		>
+			{{.locale.Tr "actions.variables.creation"}}
+		</button>
+	</div>
+</h4>
+<div class="ui attached segment">
+	{{if .Variables}}
+	<div class="ui key list">
+		{{range $i, $v := .Variables}}
+		<div class="item gt-df gt-ac gt-fw {{if gt $i 0}} gt-py-4{{end}}">
+			<div class="content gt-f1 gt-ellipsis">
+				<strong>{{$v.Title}}</strong>
+				<div class="print meta gt-ellipsis">{{$v.Content}}</div>
+			</div>
+			<div class="content">
+				<span class="color-text-light-2 gt-mr-5">
+					{{$.locale.Tr "settings.added_on" (DateTime "short" $v.CreatedUnix) | Safe}}
+				</span>
+				<button class="ui button button-ghost show-modal show-variable-edit-modal gt-p-3"
+					data-is-new="false"
+					data-old-title="{{$v.Title}}"
+					data-old-content="{{$v.Content}}"
+					data-modal="#edit-variable-modal"
+					data-base-action="{{$.Link}}/{{$v.ID}}/edit"
+					data-modal-header="{{$.locale.Tr "actions.variables.edit"}}"
+					data-tooltip-content="{{$.locale.Tr "actions.variables.edit_tooltip" $v.Content}}"
+				>
+					{{svg "octicon-pencil"}}
+				</button>
+				<button class="ui button button-ghost delete-button gt-p-3"
```

Comment:
It can use the newly introduced "link-action" confirm approach, no need to copy the "delete modal" dialog again.